spec/features/admin/merchants_index_spec.rb

```diff
@@ -54,5 +130,15 @@
 
       expect(page).to_not have_button('Disable')
     end
+
+    it 'I cannot see an enable button on merchants index' do
+      default_user = create(:user, role: 0)
+
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(default_user)
+
+      visit '/merchants'
+
+      expect(page).to_not have_button('Enable')
+    end
```

Comment:
I like it. We might also add a test to make sure a merchant-type user can't see the button either but not critical.

Reply:
Yes! Just added tests in for merchant user. Appreciate the feedback!